CONTRIBUTING.rst

```diff
@@ -0,0 +1,264 @@
+Development Setup
+=================
+
+Cloning the project
+-------------------
+
+::
+
+    git clone git@github.com:SFDO-Tooling/MetaShare
+    cd MetaShare
+
+Docker-based development (preferred)
+------------------------------------
+
+1. Install `Docker Desktop (Community Edition)`_ and make sure it is running.
+
+2. Create an ``.env`` file with the required environment variables::
+
+    cp env.docker.example .env
+
+   Edit this file to change ``DJANGO_SECRET_KEY`` and ``DJANGO_HASHID_SALT`` to
+   any two different arbitrary string values. Also set ``DB_ENCRYPTION_KEY``::
+
+    python
+    from cryptography.fernet import Fernet
+    Fernet.generate_key()
+
+   This will output a bytestring, e.g. ``b'mystring='``. Copy only the contents
+   of ``'...'``, e.g. ``DB_ENCRYPTION_KEY="mystring="``.
+
+   Finally, set the following environment variables (if you're an OddBird, you
+   can find these values in the shared Keybase team folder --
+   ``metashare/env``)::
+
+    BUCKETEER_AWS_ACCESS_KEY_ID=...
+    BUCKETEER_AWS_SECRET_ACCESS_KEY=...
+    BUCKETEER_BUCKET_NAME=...
+
+3. Run ``docker-compose build`` to build/re-build all the container images.
+
+4. Run ``docker-compose up`` to start the server(s).
+
+5. Visit `<http://localhost:8080/>`_ in your browser.
```

Comment:
@wlonk This currently isn't working for me -- maybe a port forwarding problem?

Reply:
Yep, I'm building and working on that right now.